core/src/main/java/org/javarosa/core/util/externalizable/Hasher.java

```diff
@@ -17,4 +17,5 @@
  */
 public interface Hasher {
     public byte[] getClassHashValue(Class type);
+    public int getHashSize();
```

Comment:
method declarations in public interfaces don't need to be marked as public, since it doesn't make sense to declare a private method in an interface.